accounts/views.py

```diff
@@ -202,6 +209,15 @@ def complete_profile(request):
 
     return render(request, 'accounts/volunteer_profile.html')
 
+def ajax_volunteer_rollcheck(request):
+    roll_no = request.GET.get('roll')
+    duplicate_rollcheck = Volunteer.objects.filter(roll_no=roll_no)
+    data = {}
+    if duplicate_rollcheck.exists():
+        data['isExist'] = True
+    else:
+        data['isExist'] = False
```

Comment:
You can write these in ternary form which will shorten the code above to a single line.